enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
+  gitRawCommitsOpts: { grep: "(sdk)" },
```

Comment:
This is the way to filter the only scope we want. [See more.](https://github.com/metabase/metabase/issues/42749#issuecomment-2137767536)